web/src/hooks/arrows.ts

```diff
@@ -55,22 +55,22 @@ export function filterExchanges(
 }
 
 export function useExchangeArrowsData(): ExchangeArrowData[] {
-  const [selectedDatetime] = useAtom(selectedDatetimeIndexAtom);
-  const [viewMode] = useAtom(spatialAggregateAtom);
+  const selectedDatetimeString = useAtomValue(selectedDatetimeStringAtom);
+  const viewMode = useAtomValue(spatialAggregateAtom);
   const { data } = useGetState();
 
   // Find outages in state data and hide exports from those zones
   const zonesWithOutages = useMemo(() => {
-    const zoneData = data?.data?.datetimes?.[selectedDatetime?.datetimeString]?.z;
+    const zoneData = data?.data?.datetimes?.[selectedDatetimeString]?.z;
     return zoneData
       ? Object.entries(zoneData)
           .filter(([_, value]) => value.o)
           .map(([zone]) => zone)
       : [];
-  }, [data, selectedDatetime]);
+  }, [data?.data?.datetimes, selectedDatetimeString]);
```

Comment:
Why can we change it to be `data?.data?.datetimes` instead of `data` as it was before?

Reply:
Sure but this was a ESLint rule that made the suggestion/change. I'll revert it and then we can look into that separately.